wave_vision/tests/descriptor_tests/brisk_tests.cpp

```diff
@@ -0,0 +1,197 @@
+// C++ Headers
+#include <algorithm>
+
+// Libwave Headers
+#include "wave/wave_test.hpp"
+#include "wave/vision/descriptor/brisk_descriptor.hpp"
+#include "wave/vision/detector/fast_detector.hpp"
+
+/** The wave namespace */
```

Comment:
I think that's valuable for showing what's closing, since (unlike here) the name is far out of sight. Fancy modern IDEs make it less necessary of course.

I do think `}  // end of namespace wave` is redundant, because `}` means "end of". `// namespace wave` just as recommended in [google guide](https://google.github.io/styleguide/cppguide.html#Namespaces) seems good.